Automattic/jetpack
# push-to-mirrors: Local testing should not require a GH token

#### Changes proposed in this Pull Request:

When testing locally, you should be able to run the script supplying only BUILD_BASE, GITHUB_ACTOR, and GITHUB_REF. There should be no need to set API_TOKEN_GITHUB.

#### Other information:


#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Download and unpack the build artifact, make sure `CI` is **not** set in your environment, and try running `BUILD_BASE=/path/to/build/artifact GITHUB_ACTOR=me GITHUB_REF=refs/heads/trunk projects/github-actions/push-to-mirrors/push-to-mirrors.sh`. It should succeed with a dry run.
* Merge this, then make sure the Actions trunk build pushes to mirrors successfully.
